articles/purview/tutorial-using-python-sdk.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "How to use Azure Purview Python SDK"
+description: This tutorial describes how to use the Azure Purview Python SDK to scan data and search the catalog.
+author: raoufaliouat
+ms.author: raoufaliouat
+ms.service: purview
+ms.subservice: purview-data-scanning
+ms.topic: tutorial
+ms.date: 01/10/2022
+
+# Customer intent: I can use the scanning and catalog Python SDKs to perform CRUD operations on data sources and scans, trigger scans and also to search the catalog.
+---
+
+
+
+# Tutorial: Use the Purview Python SDK
+
+The aim of this tutorial is to introduce you to the use of the Azure Purview Python SDK to programmatically do the most common operations you can find in Azure Purview Studio.
+
+In this tutorial, you will learn how to:
+
+*   Grant the required rights to work programmatically with Purview
+*   Register a Blob Storage container as a data source in Purview
+*   Define and run a scan
+*   Search the catalog
+*   Delete a data source
+
+## Prerequisites
+
+For this tutorial, you will need :
+*   Python 3.6+
+*   An active Azure Subscription 
+*   An Azure Active Directory tenant associated with you subscription
+*   An Azure storage account
+*   A Purview account
+
+
+## Give Purview access to the Storage account
+
+Before being able to scan the content of the Storage account, you need to give purview the right role. 
+Go to your Storage Account via the portal. Select Access Control (IAM). Click on the Add button and select “Add role assignment”.
+
+:::image type="content" source="media/tutorial-using-python-sdk/add-role-assignment-storage.png" alt-text="Screenshot that shows Access .Control menu of the Storage Account"::: 
+
+In the next window, search for the “Storage blob Reader” role and select it:
+
+:::image type="content" source="media/tutorial-using-python-sdk/storage-blob-reader-role.png" alt-text="Screenshot that shows the Storage Blob Reader role in the list of available roles"::: 
+
+Then go on the “Members” tab and click on “Select members”:
```

Comment:
```suggestion
Then go on the “Members” tab and select “Select members”:
```

Use 'Select' rather than 'click.'
You had already started doing this, so I'd suggest just doing a search for any rogue "click"s in your article.